src/main/java/in/casekartin/exception/ServiceException.java

```diff
@@ -0,0 +1,13 @@
+package in.casekartin.exception;
+
+public class ServiceException extends Exception {
+
+	public ServiceException(String message, NumberException e) {
```

Comment:
Change NumberException to generic one , may be need to pass more than one type of exception